src/todos.js

```diff
@@ -0,0 +1,37 @@
+let express = require('express');
+let bodyParser = require('body-parser');
+
+let app = express();
+app.use(bodyParser.json());
+
+let todos = [{id: 'jkhsdjkf', content: 'review this code'}];
```

Comment:
1. Use `id`s as key to ensure they are unique.
2. Maybe, consider using an actual database if you want data to survive server restarts.